java/src/jmri/jmrit/beantable/LogixNGTableTableAction.java

```diff
@@ -159,7 +176,30 @@ protected String getBeanText(NamedTable bean) {
 
     @Override
     protected String helpTarget() {
-        return "package.jmri.jmrit.logixng.LogixNGTableTable";  // NOI18N
+        return "package.jmri.jmrit.beantable.LogixNGTableTable";  // NOI18N
+    }
+
+    private JButton createFileChooser() {
```

Comment:
It would be good if you added the file chooser button to the `TableEditor` as well. The table editor has now the possibility to change the csv file name, but it has not the file chooser button.

If the user changes the csv file name, that change will not take effect unless the user stores the panel and restarts JMRI. CSV tables are treated as read only data and is not currently synchronized, so changing the table while LogixNG is running might cause strange errors.

Also, if the user changes the CSV table on disk, the data of the table is not read by JMRI until JMRI is restarted and the panel is loaded.

This is probably something that should be in the documentation. And maybe a message dialog when the user has changed the csv file and click on the `OK` button, that the csv file name is changed but that will not take effect unless the user stores the panel and restarts JMRI.

Reply:
This will be a TODO item.